### ISSUE TITLE
Kodi not loading 

### ISSUE THREAD

**Issue by 5kedone** [krishenriksen/AnberPorts#23]
Hi I have updated everything as of writing this post, and then installed kodi,  but the app won't load,  anything I'm missing

**Comment by iCheetle** [comment]
same here

**Comment by krishenriksen** [comment]
Can you provide "log.txt" located in the kodi directory.

**Comment by 5kedone** [comment]
Sure what's the path good sir , I check /ports / kodi and found zero logs

**Comment by oblende** [comment]
So I got Kodi to work. The problem is that packages needed by kodi are not installed by default in ArkOS. It needs python2 and curl. Python2 for necessary support, curl so controls work. Also the Kodi.sh script file doesn't start with a #! . It seems to be necessary for proper execution. This leads to the other problem that seems to be all kodi's fault, but they refuse to fix. The screen orientation for Kodi is in portrait orientation, not landscape. I've found no information on how to fix this problem.

**Comment by krishenriksen** [comment]
Updated kodi with missing shebang line
Added custom rotating fix.

**Comment by 5kedone** [comment]
I tried that just now and still don't work , this time it gets as far as showing kodi splash then quits back to ports menu

**Comment by krishenriksen** [comment]
And log.txt ?

**Comment by 5kedone** [comment]
Here you go it actually made one this time https://pastebin.com/3UXwAFgH

**Comment by oblende** [comment]
Yeah, I had worked through this error when I originally posted. It's missing python2 which is needed by kodi. As a workaround you can enable services and log into your device using ssh. Just apt install python2. After that you will have a problem that your controls won't work. It needs curl as previously mentioned so you should apt install that as well.

**Comment by uv54633** [comment]
I am also experiencing the same problem with Kodi, here is my log file.
[log.txt](https://github.com/krishenriksen/AnberPorts/files/6004957/log.txt)

**Comment by 5kedone** [comment]
Has this been fixed yet in latest versions or still a issue

**Comment by 5kedone** [comment]
Yep still not working checked again today got to menu in it once after following above commands then it just crashed after selecting any thing
